blog/2015/03/getting-to-know-matt-davis.html

```diff
@@ -0,0 +1,83 @@
++---
++layout: blog
++root: ../../..
++author: Amy Brown
++title: "Getting to Know: Matt Davis"
++date: 2015-03-31
++time: "19:30:00"
++category: ["Community"]
++---
+<!--start excerpt-->
+<p>
+This is the second in our "Getting to Know" series of contributor profiles. The subject of today's post is <a href="http://software-carpentry.org/pages/team.html#davis.m">Matt Davis</a>, a long-time Software Carpentry team member. Matt is the vice-chair of our Steering Committee, and the Software Carpentry Foundation liason for the Lesson Organization and Development subcommittee.
+</p>
+<!--end excerpt-->
+
+<h3>How did you become involved in SWC?</h3>
+<p>
+In January 2012 I enrolled in a workshop Greg Wilson did at the
+Space Telescope Science Institute.
+I ended up helping at that workshop and guiding some exercises
+for the class afterwards.
+A month later Greg invited me to Toronto to teach Python and help
+at a workshop there. I've been teaching for SWC ever since!
+</p>
+
+<h3>What is your most memorable SWC moment?</h3>
+<p>
+There have been lots of adventures with SWC.
+On my way to my first workshop in Toronto I missed my flight and
+made it to the workshop as my lesson was starting.
+I once taught an entire two-day workshop solo (plus one helper).
+(I'm not keen to do it again.)
+And I'll always remember the times I've taught at the
+Lawrence Berkeley Lab and stayed in their guest hostel
+with a beautiful view of the San Francisco Bay.
+</p>
+
+<h3>If three phrases/words described you, what would they be?</h3>
+<p>
+Chill, outside, California
+</p>
+
+<h3>What research or work do you do?</h3>
+<p>
+At my current job I write Python software used in urban planning and modeling.
+(So lots of orchestrating data flowing through statistical models.)
+At previous jobs I've worked on web applications, astronomy data pipelines
+and observation planning tools, and on data analysis/visualization
+in atmospheric science.
+</p>
+
+<h3>What programming languages and other technology do you regularly use?</h3>
+<p>
+I use the scientific Python stack
+(numpy, scipy, Pandas, matplotlib) for almost everything,
```

Comment:
@amyrbrown Could you replace "numpy, scipy" with "NumPy, SciPy"? I'm very picky about software names. =)